### ISSUE TITLE
What are the Influxdb metrics?

### ISSUE THREAD

**Issue by sereysethy** [google/cadvisor#2571]
Hi,
I use influxdb as storage from cadvisor, and I only see the following measurements.

Before I use prometheus to query metrics and save them to influxdb, they are quite different, the one with prometheus are more rich.

My question is that are the tables that I get in my influxdb correct?

```sql
> show measurements
name: measurements
name
----
cpu_usage_per_cpu
cpu_usage_system
cpu_usage_total
cpu_usage_user
fs_limit
fs_usage
load_average
memory_usage
memory_working_set
rx_bytes
rx_errors
tx_bytes
tx_errors
```

I supposed `rx` and `tx` are network `receive` and `transmit`?

**Comment by dashpole** [comment]
yes, they are receive and transmit.  The prometheus metrics tend to be the most complete, as we generally only add metrics to the storage driver-based metrics when they are requested.

By the way, what did you use to query prometheus and send to influxdb?  Were you using the prometheus server as the intermediary, or something else?

**Comment by sereysethy** [comment]
I have two different configurations, just for testing for now:

- cadvisor - prometheus - influxdb
- cadvisor - influxdb

I use the second one because I though I would almost obtain real time data
but I think there are some delay before data can be pushed to influxdb.

So the data that I can get from cadvisor are the ones that I showed in my
previous post?

Do you think if I can use the web interface api, will I get almost real
time data?

Please advise, I want to get the most recent stats for of docker containers
as fast as I can. What is a proper way to do it?

Thanks.

Le mer. 3 juin 2020 à 22:32, David Ashpole <notifications@github.com> a
écrit :

**Comment by dashpole** [comment]
cAdvisor collects metrics asynchronously, based on the `--housekeeping_interval`.  It exports metrics based on the storage_driver_buffer_duration flag.  A shorter interval will make it send more often.

**Comment by sereysethy** [comment]
I saw those options too but I only modified the
storage_driver_buffer_duration and I didn’t see any noticeable change. I
will test them both together.

Is there a way to tell cadvisor to only monitor a particular container?

Le mer. 3 juin 2020 à 23:37, David Ashpole <notifications@github.com> a
écrit :

**Comment by dashpole** [comment]
Also, what is your definition of "almost real time"  1 second, 10 seconds, 1 minute?

**Comment by dashpole** [comment]
There isn't a way to monitor only one contianer

**Comment by sereysethy** [comment]
Yes almost a second or less than a second.

Le mer. 3 juin 2020 à 23:53, David Ashpole <notifications@github.com> a
écrit :

**Comment by dims** [comment]
Closing this issue as it has been open since before January 1, 2024. If this is still relevant, please feel free to reopen or create a new issue.
